jdi-light/src/main/java/com/epam/jdi/light/asserts/core/DataListAssert.java

```diff
@@ -28,85 +29,93 @@
         element.refresh();
         return element.asData();
     }
-
+    
     /**
      * Check that all elements meet condition
+     *
      * @param condition to compare
      * @return DataListAssert
      */
     @JDIAction("Assert that each of '{name}' elements meet condition")
-    public DataListAssert<T, D> each(JFunc1<D, Boolean> condition) {
-        jdiAssert(LinqUtils.all(data(), condition::execute), Matchers.is(true));
+    public DataListAssert<T, D> each(JFunc1<D, Boolean> condition, String... messages) {
+        jdiAssert(LinqUtils.all(data(), condition::execute), Matchers.is(true), messages);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)